CHANGELOG.md

```diff
@@ -7,8 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
-- Bump minimum supported Rust version to `1.61.0`
+- Bump minimum supported Rust version to `1.65.0`
 - Change `ClipboardProvider::set_contents` parameter type to `AsRef<str>`
+- `smithay-clipboard` was bumped to `0.7.0` using the new wayland-rs stack
+- `x11_clipboard` wasa bumped to `0.8.1`
```

Comment:
I'm not sure this is relevant to endusers

Reply:
wasa